### PR TITLE
Database

### PR DESCRIPTION
**Pull request by camilleldn**

### CODE REVIEW

**Comment by camilleldn** [comment]
So what we are missing here is a 'nicer' database setup: we are currently hardocding 'password' as the password and the database name. This is all things that should be in the environmet down the line. Worth having a chat with Andrew H. ?

**Comment by camilleldn** [comment]
We also want probably want to mock the database call method in the 404 test case, so that database doesn't get hit during our unit tests?

File: service/server.py [hunk 1]
```diff
@@ -4,109 +4,27 @@
...
-        json_file = open('service/DT160760.json', 'r')
-        sample_json = json_file.read()
-        data = json.loads(sample_json)
-        return jsonify(data)
+    data = get_title_register(title_ref)
+    if data:
+        result = {
+          "data": data.register_data,
+          "title_number": data.title_number,
+          "geometry_data": data.geometry_data,
```
Comment (nickgravgaard) [inline service/server.py]: This is only indented by 2 spaces.

File: tests/test_app.py [hunk 2]
```diff
@@ -1,17 +1,39 @@
...
 import json
 
 import mock
-import unittest
 import requests
 import responses
 
+TitleRegisterData = namedtuple('TitleRegisterData',
+  ['title_number', 'register_data', 'geometry_data']
+)
```
Comment (nickgravgaard) [inline tests/test_app.py]: Another indentation problem...

File: tests/test_app.py [hunk 3]
```diff
@@ -1,17 +1,39 @@
...
+
+    @mock.patch('service.server.get_title_register', return_value=DN1000_title)
+    def test_get_valid_title_path(self, mock_data):
+        response = self.app.get('/titles/DN1000')
+        assert response.status_code == 200
+
+    @mock.patch('service.server.get_title_register', return_value=DN1000_title)
+    def test_get_correct_title_number(self, mock_data):
+        response = self.app.get('/titles/DN1000')
+        self.assertTrue(str('"title_number": "DN1000"') in str(response.data))
```
Comment (nickgravgaard) [inline tests/test_app.py]: No need to str() a string

**Pull request merged**